samples/Directory.Build.props

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <IsTestGenerationSrcProject Condition="!$(MSBuildProjectName.EndsWith('.Tests'))">true</IsTestGenerationSrcProject>
+    <IsMgmtLibrary Condition="!$(MSBuildProjectName.EndsWith('.Tests')) and ($(MSBuildProjectName.StartsWith('_Azure.ResourceManager')) or $(MSBuildProjectName.StartsWith('Mgmt')))">true</IsMgmtLibrary>
```

Comment:
The CI is failing due to `AzureSample.ResourceManager.Storage` doesn't follow the naming check here.
Consider to update to `$(MSBuildProjectName.Contains('.ResourceManager')) or $(MSBuildProjectName.Contains('Mgmt'))`

Reply:
Yes. I will update. Thanks